src/tilemaker/entrypoint.sh

```diff
@@ -7,6 +7,8 @@ source /opt/pfb/tilemaker/scripts/utils.sh
 
 export TL_MIN_ZOOM="${TL_MIN_ZOOM:-8}"
 export TL_MAX_ZOOM="${TL_MAX_ZOOM:-18}"
+export TL_NUM_PARTS="${TL_NUM_PARTS:-4}"
```

Comment:
Or we could just set `vcpus` to 4 so that having different-sized jobs wouldn't cause idle CPU time because they'd always come in sets of 4 roughly-equal jobs.